src/gene_demulti/scSplit/script.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# add additional params
+extra_params=( )
+
+if [ ! -z "$par_sub" ]; then
+  extra_params+=( "--sub $par_sub" )
+fi
+  
+if [ ! -z "$par_ems" ]; then
+  extra_params+=( "--ems $par_ems" )
+fi
+
+if [ ! -z "$par_dbl" ]; then
+  extra_params+=( "--dbl $par_dbl" )
+fi
+
+if [ ! -z "$par_vcf_known" ] ; then
+  extra_params+=( "--vcf $par_vcf_known" )
+fi
+
+if [ ! -d "$par_output" ]; then
+  mkdir $par_output
+fi
+
+scSplit_loc="/usr/local/lib/python3.9/site-packages/scSplit"
```

Comment:
`scSplit` has a really unconventional way to install. I would really like this path to be python version agnostic. Could you install `scSplit` using the docker `run` directive (e.g. download it using wget and copying the script to `/usr/local/bin` or `/usr/bin`)